bindings_generator/wrapper.h

```diff
@@ -13,4 +13,4 @@ typedef enum IRReflectionVersion IRReflectionVersion;
 // Undef __APPLE__ so we avoid generating bindings for dispatch/dispatch.h
 #undef __APPLE__
 
-#include <metal_irconverter/metal_irconverter.h>
+#include "metal_irconverter/metal_irconverter.h"
```

Comment:
Fwiw you are no longer manually changing this file, right? Maybe we should remove this comment in `wrapper.h`:

```c
// This wrapper #includes the metal_irconverter header with a few tweaks to make bindgen happy
```